src/main/resources/config/application.properties.sample

```diff
@@ -14,24 +14,34 @@ spring.thymeleaf.mode=HTML5
 spring.thymeleaf.encoding=UTF-8
 spring.thymeleaf.content-type=text/html
 
```

Comment:
This comment applies to ` spring.thymeleaf.view-names= MailTemplate, loginForm`. This line can be removed as well (it only makes stuff complicated). I already did this in the neon-admin branch, but I guess you can do this in this PR before that is merged.